### ISSUE TITLE
Questions about the third phase of training

### ISSUE THREAD

**Issue by lshy687** [iLampard/ep_llm#8]
Hello, I have a question regarding Step 3. Is the training data in this step derived from the results of the initial test set after inference by the LLM? Thank you for your clarification.

**Comment by iLampard** [comment]
Hi, 

the training data at step 3 is still derived from the train set in step 1.
